contracts/scripts/native_solc_compile_all_ccip

```diff
@@ -44,6 +44,10 @@ compileContract () {
     echo "CCIPReaderTester uses 1 optimizer runs for reduced contract size."
       optimize_runs=1
       ;;
+    "ccip/test/helpers/USDCReaderTester.sol")
+    echo "USDCReaderTest uses 1 optimizer runs for reduced contract size."
```

Comment:
Why? It would probably fit perfectly well without this custom logic?

Reply:
It was a blind copy paste from other tests, can remove

Reply:
imo the other one also doesn't need it.

Reply:
Removed this one and the previous one